keyboards/yampad/yampad.c

```diff
@@ -0,0 +1,64 @@
+    
+/* Copyright 2019
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include "yampad.h"
+
+#if defined(OLED_DRIVER_ENABLE)
+__attribute__((weak))
+oled_rotation_t oled_init_user(oled_rotation_t rotation) {
+    return OLED_ROTATION_270;  // flips the display 270 degrees
+}
+
+__attribute__((weak)) 
+void oled_task_user(void) {
+  // Host Keyboard Layer Status
+  oled_write_P(PSTR("Layer"), false);
+  switch (get_highest_layer(layer_state)) {
+    case _BL:
+      oled_write_ln_P(PSTR(" BAS"), false);
+      break;
+    case _NV:
+      oled_write_ln_P(PSTR(" NAV"), false);
+      break;
+    case _FN:
+      oled_write_ln_P(PSTR(" RGB"), false);
+      break;
```

Comment:
Neither of the keymaps can actually be compiled with OLED enabled because the layer references used here don't exist until the keymaps are processed.

Unfortunately I don't have any experience with OLED nor skill at programming, so I don't know how to fix this.

Reply:
Yup, this is correct.  
The best way is just to put in the numbers, not the names. 

Otherwise, you'd want to add a enum for them in the header. But this can cause issues for users.  So ... just the number is better.